3.9/installation-mac-osx.md

```diff
@@ -12,6 +12,12 @@ You can install ArangoDB on macOS in different ways:
 - [_DMG_ Package](#package-installation)
 - [_tar.gz_ Archive](#installing-using-the-archive)
 
+{% hint 'warning' %}
+Currently, the ArangoDB installation on macOS via Homebrew,
```

Comment:
"currently" should be avoided: https://developers.google.com/style/timeless-documentation

We can be explicit and say that 3.9 does not run on Apple's ARM chips. 3.8 and older do work via Rosetta 2 emulation, and 3.10 will have native ARM support.